aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/CallbackContext.java

```diff
@@ -7,6 +7,13 @@
 @lombok.ToString
 @lombok.EqualsAndHashCode(callSuper = true)
 public class CallbackContext extends StdCallbackContext {
+    final String ARN_TEMPLATE = "arn:aws:rds:%s:%s:pg:%s";
+
     private boolean parametersApplied;
     private String dbParameterGroupArn;
```

Comment:
Let's ensure that `Arn` is a serializable datum

Reply:
I kept it simple as String. We don't need any details back from Arn from it.